reservations/admin.py

```diff
@@ -12,28 +12,28 @@
 from pytz import timezone
 
 from .models import (
-    BerthReservation,
+    BerthApplication,
     BerthSwitchReason,
     HarborChoice,
     WinterStorageAreaChoice,
-    WinterStorageReservation,
+    WinterStorageApplication,
 )
 from .notifications import NotificationType
 from .utils import (
-    export_berth_reservations_as_xlsx,
-    export_winter_storage_reservations_as_xlsx,
+    export_berth_applications_as_xlsx,
+    export_winter_storage_applications_as_xlsx,
 )
 
 logger = logging.getLogger(__name__)
 
 
-class ReservationTypeFilter(admin.SimpleListFilter):
-    title = _("Reservation type")
+class ApplicationTypeFilter(admin.SimpleListFilter):
+    title = _("application type")
```

Comment:
nitpick: this is visible in the django admin and looks more official when starts with a capital letter